### ISSUE TITLE
Are the pertained weights frozen while training?

### ISSUE THREAD

**Issue by prateekmalhotra-hover** [ternaus/TernausNet#15]
Hi! Great work! I just wanted to inquire in more detail whether, while training, you're freezing the old weights. Thanks!

**Comment by iperov** [comment]
of course no

**Comment by godsmokescrack** [comment]
I think the pretrained weights should be frozen (at least at first) or some of the advantage they give you will be erased.  For instance, the images below show training/validation loss curves (on some small segmentation dataset) for three different scenarios.  The best performance was given by freezing the pretrained knowledge.

1) Frozen weights on pretrained vgg11 encoder:
![ternaus_frozen](https://user-images.githubusercontent.com/79334220/108853189-6bea1c80-75b4-11eb-94d8-f2fa9cf9a5fc.png)

2) Unfrozen weights on pretrained encoder:
![DRIVE_ternausnet_loss_curves](https://user-images.githubusercontent.com/79334220/108853229-79070b80-75b4-11eb-991a-9b5d12014763.png)

3) Randomly initialized (pretrained=False):
![DRIVE_ternausnet_loss_curves_pt_false](https://user-images.githubusercontent.com/79334220/108853240-7c01fc00-75b4-11eb-96ba-09355c4edc8b.png)

**Comment by rrryan2016** [comment]
Has the author already provided pretrained weights?
